--- ink-0.4.0/Makefile.orig	2007-03-13 22:44:24.000000000 +0100
+++ ink-0.4.0/Makefile	2007-04-03 23:48:16.112252342 +0200
@@ -1,11 +1,12 @@
-CFLAGS = -Wall -O2 -linklevel
-DESTDIR = /usr/local/
+OPTFLAGS = -O2
+CFLAGS = $(OPTFLAGS) -Wall -linklevel
+DESTDIR =
 
 all: ink 
 
 install: all
-	mkdir -p $(DESTDIR)/bin
-	install -m 755 -o root -g root ink $(DESTDIR)/bin/ink
+	mkdir -p $(DESTDIR)/usr/bin
+	install -m 755 ink $(DESTDIR)/usr/bin/ink
 
 clean:
 	rm -fr ink *.o *~
