--- ink-0.4.1/Makefile.orig	2006-09-06 16:06:22.000000000 +0200
+++ ink-0.4.1/Makefile	2007-06-01 22:13:52.387901596 +0200
@@ -1,12 +1,12 @@
-CFLAGS = -Wall -O2 -L$(PWD)/../../../libinklevel/cvs/libinklevel \
-         -I$(PWD)/../../../libinklevel/cvs/libinklevel -linklevel
-DESTDIR = /usr/local/
+OPTFLAGS = -O2
+CFLAGS = -Wall $(OPTFLAGS) -linklevel
+DESTDIR =
 
 all: ink 
 
 install: all
-	mkdir -p $(DESTDIR)/bin
-	install -m 755 -o root -g root ink $(DESTDIR)/bin/ink
+	mkdir -p $(DESTDIR)/usr/bin
+	install -m 755 ink $(DESTDIR)/usr/bin/ink
 
 clean:
 	rm -fr ink *.o *~
