diff -aurN ink.orig/Makefile ink/Makefile
--- ink.orig/Makefile	2003-12-03 17:41:37.000000000 +0100
+++ ink/Makefile	2003-12-04 17:32:08.000000000 +0100
@@ -1,11 +1,11 @@
-CFLAGS = -Wall -O -linklevel
-DESTDIR = /usr/local/
+CFLAGS = $(OPTFLAGS) -Wall -linklevel
+#DESTDIR = /usr/local
 
 all: ink 
 
 install: all
-	mkdir -p $(DESTDIR)/bin
-	install -m 755 -o root -g root ink $(DESTDIR)/bin/ink
+	mkdir -p $(DESTDIR)/usr/bin
+	install -c ink $(DESTDIR)/usr/bin/ink
 
 clean:
 	rm ink *~
